README.md

```diff
@@ -109,13 +109,11 @@ Developers wishing to activate this functionality must pass the `--moneybadger`
 
 | SDK                                 | Minimum Version |
 | ----------------------------------- | --------------- |
-| @firebolt-js/firebolt-sdk           | 0.6.0-alpha.1   |
-| @firebolt-js/firebolt-manage-sdk    | 0.1.0-alpha.2   |
 | @firebolt-js/firebolt-discovery-sdk | 0.1.0-alpha.1   |
```

Comment:
Technically, the deleted comments about minimum versions of the SDKs are correct. But those are old enough now that I think we should just remove this section about minimum versions.

Reply:
Addressed